certs/pki/vault/vault.go

```diff
@@ -45,8 +45,6 @@ type Cert struct {
 }
 
 // Agent represents the Vault PKI interface.
-//
-//go:generate mockery --name Agent --output=../mocks --filename pki.go --quiet --note "Copyright (c) Abstract Machines"
 type Agent interface {
 	// IssueCert issues certificate on PKI
```

Comment:
Agent interface should be defined in common at `certs/agent.go`

And both Abstract Machines certs and Vault should use same Agent interface

Reply:
This is going to be a challenge because the methods in the interface are very different.

Reply:
Why we have remove the certs repo? Just trying to understand  the reason